storage/innobase/clone/clone0repl.cc

```diff
@@ -471,7 +472,14 @@ void Clone_persist_gtid::flush_gtids(THD *thd) {
     m_compression_gtid_counter = 0;
     /* Write non-innodb GTIDs before compression. */
     write_other_gtids();
-    err = gtid_table_persistor->compress(thd);
+
+    /* Compress the gtid_executed table if there is an explicit compression
+       request. Otherwise signal the gtid persister thread to perform
+       compression. */
```

Comment:
```suggestion
    /* Compress the gtid_executed table if there is an explicit compression
    request. Otherwise signal the gtid persister thread to perform
    compression. */
```

Reply:
Done.